app/mobile/lib/view/topic/long_press_dialog.dart

```diff
@@ -0,0 +1,208 @@
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:persistent_bottom_nav_bar/persistent_tab_view.dart';
+import 'package:stacked/stacked.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+Widget longPressDialog(Resource r) => ViewModelBuilder<
+        ResourcePageViewModel>.reactive(
+    viewModelBuilder: () => ResourcePageViewModel(),
+    builder: (context, viewModel, child) {
+      return Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Container(
+              margin: const EdgeInsets.all(5),
+              width: double.infinity,
+              padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 20),
+              child: Text(
+                "Discussion",
+                style: TextStyle(
+                    color: Colors.white,
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold),
+                textAlign: TextAlign.center,
+              ),
+              decoration: BoxDecoration(
+                  color: CustomColors.main,
+                  borderRadius: BorderRadius.circular(10)),
+            ),
+            Container(
+              margin: const EdgeInsets.all(5),
+              width: double.infinity,
+              padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 20),
+              child: Text(
+                "Details",
+                style: TextStyle(
+                    color: Colors.white,
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold),
+                textAlign: TextAlign.center,
+              ),
+              decoration: BoxDecoration(
+                  color: Color.fromARGB(255, 68, 84, 141),
+                  borderRadius: BorderRadius.circular(10)),
+            ),
+            GestureDetector(
+              onTap: () {
+                PersistentNavBarNavigator.pushNewScreen(context,
+                    screen: editResourceView(r));
+              },
+              child: Container(
+                margin: const EdgeInsets.all(5),
+                width: double.infinity,
+                padding:
+                    const EdgeInsets.symmetric(horizontal: 20, vertical: 20),
+                child: Text(
+                  "Edit",
+                  style: TextStyle(
+                      color: Colors.white,
+                      fontSize: 20,
+                      fontWeight: FontWeight.bold),
+                  textAlign: TextAlign.center,
+                ),
+                decoration: BoxDecoration(
+                    color: CustomColors.main,
+                    borderRadius: BorderRadius.circular(10)),
+              ),
+            ),
+            GestureDetector(
+              onTap: () {
+                showDialog(
+                  context: context,
+                  builder: ((context) => AlertDialog(
+                        content: Column(
```

Comment:
I have experienced overflow in my device. Here is a screenshot: 
<img src="https://user-images.githubusercontent.com/56507424/205573697-2f49fcc4-bc98-420a-ad15-b1afadb8ce5b.jpeg" width= 35%>

Reply:
I fixed this problem by changing the row widget to the wrap widget. It should be fine now.

Reply:
I have confirmed the new warning box. It's working :)